### PR TITLE
Feature/task4784 implement dynamo db

### PR DESCRIPTION
**Pull request by mohamed-samandi**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

A description of the change.

#### 🤔 Why

Why it's needed, background context.

#### 🛠 How

More in-depth discussion of the change or implementation.

#### 👀 Evidence

Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR

#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: java/pom.xml [hunk 1]
```diff
@@ -425,11 +386,14 @@
         <dynamodb.profile.name>dynamodb</dynamodb.profile.name>
       </properties>
       <dependencies>
+        <dependency>
```
Comment (sdavis-amido) [inline java/pom.xml:389]: Remove this as won't build with it in currently

File: java/src/main/resources/application.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
 spring:
   profiles:
     include:
-      - "@aws.profile.name@"
-      - "@azure.profile.name@"
-      - "@cosmosdb.profile.name@"
-      - "@dynamodb.profile.name@"
+      - aws
+      - azure
+      - dynamodb
```
Comment (sdavis-amido) [inline java/src/main/resources/application.yml:6]: This is wrong - this needs to be an array of "@aws....@" type properties, and not literals

**Review by sdavis-amido — CHANGES_REQUESTED** [review]

File: java/src/main/resources/application.yml (hunk 2 above)
Comment (mohamed-samandi) [reply java/src/main/resources/application.yml]: fixed

File: java/pom.xml (hunk 1 above)
Comment (mohamed-samandi) [reply java/pom.xml:389]: actually the build was succesfull  if it put inside a profile

**Review by sdavis-amido — APPROVED** [review]

File: java/deploy_scenario.sh [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+  fi
+
+}
+
+#####################
+#####################
+
+check_environment
+
+declare -a ALL_SPRING_PROFILES=(aws azure cosmosdb dynamodb servicebus kafka sqs)
```
Comment (sdavis-amido) [inline java/deploy_scenario.sh:142]: Messaging elements need removing from here (servicebus kafka sqs)

**Review by sdavis-amido — CHANGES_REQUESTED** [review]

File: java/deploy_scenario.sh (hunk 3 above)
Comment (mohamed-samandi) [reply java/deploy_scenario.sh]: fixed

**Pull request merged**